pytransform3d/batch_rotations.py

```diff
@@ -38,6 +38,52 @@ def norm_vectors(V, out=None):
     return out
 
 
+def norm_axis_angles(a):
+    """Normalize axis-angle representation.
+
+    Parameters
+    ----------
+    a : array-like, shape (..., 4)
+        Axis of rotation and rotation angle: (x, y, z, angle)
+
+    Returns
+    -------
+    a : array, shape (..., 4)
+        Axis of rotation and rotation angle: (x, y, z, angle). The length
+        of the axis vector is 1 and the angle is in [0, pi). No rotation
+        is represented by [1, 0, 0, 0].
+    """
+    a = np.asarray(a)
+
+    # Handle the case of only one axis-angle instance
+    only_one = a.ndim == 1
+    a = np.atleast_2d(a)
+
+    angles = a[..., 3]
+    norm = np.linalg.norm(a[..., :3], axis=-1)
+
+    zero_mask = (angles == 0.0) | (norm == 0.0)
```

Comment:
Maybe a more suitable name (but only my subjective opinion):

```suggestion
    no_rotation_mask = (angles == 0.0) | (norm == 0.0)
```